src/plugins/scopes/ScopesApi.js

```diff
@@ -0,0 +1,474 @@
+/* @flow */
+import type Element from '../../elements/Element';
```

Comment:
I don't see tests for this whole file, like constructor call or anything in that nature, although it is pretty much consist with private methods/properties